techhive-ui/src/components/Login/Login.jsx

```diff
@@ -27,34 +29,45 @@ const styles = {
       border: 'none',
       borderRadius: '5px',
       cursor: 'pointer',
-      transition: 'all 0.3s ease',
-      ':hover': {
-        background: '#0056b3'
-      }
+      transition: 'all 0.3s ease'
     }
   };
 
 function Login() {
   const [email, setEmail] = useState("");
   const [password, setPassword] = useState("");
+  const [error, setError] = useState(null);
 
   const navigate = useNavigate(); 
 
+  const { setUser } = useContext(UserContext);
+
   const handleSubmit = async e => {
     e.preventDefault();
     try {
-      const response = await axios.post("http://localhost:3000/auth/login", { email, password });
-      console.log(response.data);
-      navigate('/');
-    } catch (err) {
-      console.error(err.response.data); 
-    }
+        const response = await axios.post("http://localhost:3000/auth/login", { email, password });
+        localStorage.setItem('user', JSON.stringify(response.data));
+        setUser(response.data.user);
+        console.log(response.data);
+        navigate('/');
+      } catch (err) {
+        console.error(err.response.data); 
+        if(err.response.data.message === "User not found"){
+          setError("No account found. Please create an account.");
+          setTimeout(() => navigate('/signup'), 3000);
+        }
+        else {
+          // something else
```

Comment:
are you planning to implement other error messages?

We can commit code with TODO comments for future iterations, but typically we want them to be more informative than this. We also wouldn't want to submit an empty else statement. Instead you can keep just the if-statement and add an informative comment "TODO add additional error handling for X type of errors"

Reply:
Okay noted. Thank you